pkg/util/profiling/profiling.go

```diff
@@ -58,11 +57,11 @@ func Start(apiKey, site, env, service string, period time.Duration, cpuDuration
 	// If block or mutex profiling was configured via runtime configuration, pass current
 	// values to profiler. This prevents profiler from resetting mutex profile rate to the
 	// default value; and enables collection of blocking profile data if it is enabled.
-	if frac := runtime.SetMutexProfileFraction(-1); frac > 0 {
-		options = append(options, profiler.MutexProfileFraction(frac))
+	if mutexFraction > 0 {
```

Comment:
Can `mutexFraction` still be changed dynamically?

Reply:
Global go runtime profiling settings - yes, these are still exposed as runtime configuration; we manipulate these elsewhere. This piece of code here is responsible for enabling mutex profiles in `dd-trace-go/profiler` if mutex profiles were already enabled for the core agent elsewhere.